tests/core/test_streams.py

```diff
@@ -728,9 +728,13 @@ class UnselectedStream(SelectedStream):
         name = "unselected_stream"
         selected_by_default = False
 
+    class AbstractStream(RESTStream):
+        url_base = "https://example.com"
+        __abstract__ = True
+
     class MyTap(tap_class):
         def discover_streams(self):
-            return [SelectedStream(self), UnselectedStream(self)]
+            return [SelectedStream(self), UnselectedStream(self), AbstractStream(self)]
```

Comment:
Would it be possible to avoid this somehow? As in, auto-discover/instantiate any parent streams that are abstract?

Reply:
That's a very good point. I'll try to do that.